frontend/src/App.js

```diff
@@ -1,12 +1,24 @@
 import SignUp from "./pages/User/SignUp/SignUp";
 import Login from "./pages/User/Login/Login";
+import UserDashboard from "./pages/User/UserDashboard";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Admin from "./pages/Admin/Admin";
+import Unauthorized from "./pages/Unauthorized";
+import RequireAuth from "./middlewares/RequireAuth";
+
```

Comment:
separate and group imports
```suggestion
import { BrowserRouter as Router, Route, Routes } from "react-router-dom";

import SignUp from "./pages/User/SignUp/SignUp";
import Login from "./pages/User/Login/Login";
import UserDashboard from "./pages/User/UserDashboard";
import Admin from "./pages/Admin/Admin";
import Unauthorized from "./pages/Unauthorized";
import RequireAuth from "./middlewares/RequireAuth";

```